packages/vector_graphics_compiler/test/resolver_test.dart

```diff
@@ -0,0 +1,100 @@
+import 'package:test/test.dart';
+import 'package:vector_graphics_compiler/src/geometry/basic_types.dart';
+import 'package:vector_graphics_compiler/src/geometry/matrix.dart';
+import 'package:vector_graphics_compiler/src/geometry/path.dart';
+import 'package:vector_graphics_compiler/src/paint.dart';
+import 'package:vector_graphics_compiler/src/svg/node.dart';
+import 'package:vector_graphics_compiler/src/svg/parser.dart';
+import 'package:vector_graphics_compiler/src/svg/visitor.dart';
+
+void main() {
+  test(
+      'Resolves PathNodes to ResolvedPathNodes by flattening the transform '
+      'and computing bounds', () {
+    final PathNode pathNode = PathNode(
+      Path(
+        commands: const <PathCommand>[
+          MoveToCommand(0, 0),
+          MoveToCommand(10, 10),
+        ],
+      ),
+      SvgAttributes(
```

Comment:
I have a slight preference for not opening up visibility on this for testing and instead making sure we can go from a relevant SVG string to a node we can pump through the visitor correctly. 

If we do open it up for testing, we should at least mark it as `@visibileForTesting`.

Reply:
(But again, my preference is to keep this as a private implementation detail of the parser that we don't directly test)

Reply:
Yeah I have mixed feelins about this. I could just do e2e but TBH the process of debugging that transform failure earlier was nighmarish because of the massive diff